src/components/ux-tooltip/ux-tooltip.hbs

```diff
@@ -0,0 +1,2 @@
+This is just a normal text but
+<span decorator="tooltip:{{.}}" tabindex="0"><strong>this</strong></span> is a tooltip.
```

Comment:
In this case, {{content}} is actually content for inside the tooltip, not the text for the link. Anything inside the JSON is being passed straight into the decorator. Is this approach OK?